src/codelabs/async-await.md

```diff
@@ -42,6 +42,18 @@ another operation to finish. Here are some common asynchronous operations:
 To perform asynchronous operations in Dart, you can use the `Future` class
 and the `async` and `await` keywords.
 
+Typically, APIs that return `Future` objects
+fall into one of the following categories:
+
+* APIs that are backed by logic running on a different thread. 
+  For example, `File.readAsString` (and most of the other I/O APIs) 
+  call operating system functions from threads on a dedicated thread pool, 
```

Comment:
Non-native English speaker here: Should `call` be `calls` since it binds to `File.readAsString`, or does the parenthesis affect it?

Reply:
Nice catch. I believe technically the content inside the parentheses should not affect the surrounding sentence, so it should be `calls`. 

It seems a bit weird if you read the parenthetical statement though, so if we keep this text, perhaps we should just remove the parentheses.